### PR TITLE
Feat: Hero Theming

### PR DESCRIPTION
**Pull request by renatamottam**
## What's the purpose of this pull request?
This PR applies the new Theming structure #407 to `Hero`: new scoped tokens linked to global ones.

## How does it work?
This PR uses local variables to parameterize the `Hero` properties, then connects these scoped tokens to the global ones.

### Primary
||Before|After|
|-|-|-|
|Mobile|image|image|
|Desktop|image|image|

### Secondary
||Before|After|
|-|-|-|
|Mobile|image|image|
|Desktop|image|image|

### New global tokens
I've proposed a new text size scale for mobile:
`--fs-text-scale-mobile               : 1.13;`

### New local tokens for `Hero`
|Local token|Default value / global token linked|
|-|-|
 | `--fs-hero-bkg-color`                  | `var(--fs-color-primary-bkg)` |
 | `--fs-hero-title-line`                 | `1.1` |
 | `--fs-hero-text-color`                 | `var(--fs-color-text-inverse)` |
 | `--fs-hero-text-size`                  | `var(--fs-text-size-lead)` |
 | `--fs-hero-text-line`                  | `1.33` |
 | `--fs-hero-primary-img-height-mob`     | `#{rem(240px)}` |
 | `--fs-hero-primary-img-height-desk`    | `#{rem(464px)}` |
 | `--fs-hero-primary-title-size`         | `var(--fs-text-size-7)` |
 | `--fs-hero-secondary-img-height-mob`   | `#{rem(180px)}` |
 | `--fs-hero-secondary-img-height-desk`  | `#{rem(227px)}` |
 | `--fs-hero-secondary-title-size`       | `var(--fs-text-size-6)` |

## How to test it?
The `Hero` component should look very similar as it was before this effort.

## Checklist
- [x] CHANGELOG entry added

### CODE REVIEW

**Comment by sergiofontes** [comment]
Great work, miga!

I'm wondering, though, if—for the sake of conciseness and to ease the tokens' findability through the browser's Inspect—it's worth shrinking a little bit these local tokens.

For instance: 
*  ~`--fs-hero-text-title-size`~ could be `--fs-hero-title-size`;
* and ~`--fs-hero-text-paragraph-size`~ could simply be `--fs-hero-text-size`.

What do you think?

File: src/components/ui/Hero/hero.scss [hunk 1]
```diff
@@ -1,10 +1,30 @@
...
+  /* stylelint-disable */
+
+  // Design Tokens for Hero Section
+  // --------------------------------------------------------
+
+  // Default properties
+  --fs-hero-bkg-color          : var(--fs-color-primary-bkg);
+
+  --fs-hero-title-size         : var(--fs-text-size-7);
+  --fs-hero-title-small-size   : var(--fs-text-size-6);
```
Comment (sergiofontes) [inline src/components/ui/Hero/hero.scss:13]: `-small` should be placed at the end: `--fs-hero-title-size-small`

**Review by sergiofontes — CHANGES_REQUESTED** [review]

**Review by hellofanny — APPROVED** [review]
Great work, miga! 
I'm not sure about the type variants renaming `type Variant = 'primary' | 'secondary'` because it's not very clear what does it mean, but fill the purpose and I don't have any better suggestion 🙈

**Review by eduardoformiga — COMMENTED** [review]
LGTM! 👏 
I notice some visual differences compared with the mobile/desktop versions from the https://base.vtex.app/ in both variations. Was that intentional?

**Review by eduardoformiga — APPROVED** [review]

**Review by saranicoly — APPROVED** [review]
LGTM!

**Review by ArthurTriis1 — APPROVED** [review]

**Review by lucasfp13 — APPROVED** [review]

**Pull request merged**